armi/reactor/blocks.py

```diff
@@ -437,18 +437,22 @@ def getMicroSuffix(self):
         setting has length 1 (i.e. no burnup groups are defined). This is useful for
         high-fidelity XS modeling of V&V models such as the ZPPRs.
         """
-
         bu = self.p.buGroup
         if not bu:
             raise RuntimeError(
                 "Cannot get MicroXS suffix because {0} in {1} does not have a burnup group"
                 "".format(self, self.parent)
             )
+
         xsType = self.p.xsType
-        if len(xsType) == 2 and len(settings.getMasterCs()["buGroups"]) == 1:
-            return xsType
-        else:
+        if len(xsType) == 1:
             return xsType + bu
+        elif len(xsType) == 2 and ord(bu) < ord("A"):
```

Comment:
The `<` should actually be `>`, I think?

That was my mistake in the original suggestion.

Reply:
Updated. And watching the unit tests.

And considering maybe we need more of them.